src/Disks/DiskBackup.cpp

```diff
@@ -0,0 +1,75 @@
+#include "DiskBackup.h"
+
+#include <Disks/DiskFactory.h>
+#include "Disks/IDisk.h"
+#include <Disks/ObjectStorages/MetadataStorageFactory.h>
+
+
+namespace DB
+{
+
+DiskBackup::DiskBackup(const String & name_, DiskPtr delegate_, MetadataStoragePtr metadata_) : IDisk(name_), delegate(delegate_), metadata(metadata_)
```

Comment:
Could you please add some documentation to https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/operations/storing-data.md about this disk? What its usecase, how to configure it and maybe some small usage example.